### ISSUE TITLE
segmentation details

### ISSUE THREAD

**Issue by pawopawo** [Belval/MobileNetV3#5]
The number of the last block channel of mobilenetv3 in the paper is reduced by a factor of 2, and atrous conv is used

**Comment by Belval** [comment]
Yes, I did diverge from the original paper on that point because otherwise, the pooling layer did not have the right size.

I think this is due to a mistake somewhere but I could not find it. Please tell me if you do.
